From b8ce2a3e5ff8d62d4ebba92c1b7bb885d5a707db Mon Sep 17 00:00:00 2001
From: Alexander Alemayhu <alexander@alemayhu.com>
Date: Tue, 18 Oct 2016 11:22:14 +0200
Subject: [PATCH 2/3] staging: rtl8192e: remove redundant assignment

'pHTInfo->CurrentMPDUDensity = pHTInfo->CurrentMPDUDensity;' does
nothing.

Discovered using coccinelle and changed with following script:

@@
expression x;
@@

- x = x;

Signed-off-by: Alexander Alemayhu <alexander@alemayhu.com>
---
 drivers/staging/rtl8192e/rtl819x_HTProc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/staging/rtl8192e/rtl819x_HTProc.c b/drivers/staging/rtl8192e/rtl819x_HTProc.c
index dd9c0c868361..2a3431115edb 100644
--- a/drivers/staging/rtl8192e/rtl819x_HTProc.c
+++ b/drivers/staging/rtl8192e/rtl819x_HTProc.c
@@ -821,8 +821,6 @@ void HTUseDefaultSetting(struct rtllib_device *ieee)
 		pHTInfo->bCurrentAMPDUEnable = pHTInfo->bAMPDUEnable;
 		pHTInfo->CurrentAMPDUFactor = pHTInfo->AMPDU_Factor;
 
-		pHTInfo->CurrentMPDUDensity = pHTInfo->CurrentMPDUDensity;
-
 		HTFilterMCSRate(ieee, ieee->Regdot11TxHTOperationalRateSet,
 				ieee->dot11HTOperationalRateSet);
 		ieee->HTHighestOperaRate = HTGetHighestMCSRate(ieee,
-- 
2.10.1

